packages/build/src/barque.spec.ts

```diff
@@ -6,6 +6,7 @@ import Config from './config';
 import sinon from 'sinon';
 import fs from 'fs-extra';
 import path from 'path';
+import { URL } from 'url';
```

Comment:
TypeScript has problems understanding that `URL` is a global in Node.js, which is a bit surprising, but I also don’t think there’s any harm in importing it :woman_shrugging: